demos/getting_started.md

```diff
@@ -0,0 +1,146 @@
+# YesWorkflow Web Components Getting Started Guide
+
+## Prerequisites: 
+### Required:
+* Python 3.6 or greater
+* Pip for the installed Python3 environment
+* [Clone the YesWorkflow Web Components Repository](https://github.com/aniehuser/senior-design-group10)
+* Install project dependencies by executing `pip install -r requirements.txt`
+
+### Optional:
+The YesWorkflow Web Components Repository comes with an already packaged YesWorkflow Jar file that can be used for the purposes of saving one's work to the website.
+
+If one would like to see the source code for the CLI Java tool it can be found at the [YesWorkflow Prototypes Repository](https://github.com/aniehuser/yw-prototypes).
+
+For more detailed instruction on how to begin using and developing the CLI please see the README found within this repository.
+
+## Installation:
+After installing all prerequisites navigate to the folder in the cloned website repository labeled scripts. To quickly launch the website execute *deploy_python3.sh* if on MacOs or Linux. If on a Window's environment execute *deploy_python.sh*. Note that these two files are simply a collection of bash commands that will initialize and begin running the YesWorkflow Web Components website. 
+
+If unable to run these bash files, the Python3 commands inside can be executed separately through the terminal.
+
+For documentation on the commands found in these files please see the [Django documentation](https://docs.djangoproject.com/en/2.1/).
+
+## Key Terms:
+* Workflows
+* Versions
+* Runs
+* Tags
+
+## CLI Tool
+### YesWorkflow Save Command and Configurations
```

Comment:
We're going to want to update this from the updated configurations documenation. It might be better to leave a link so we dont have to update both versions https://github.com/aniehuser/senior-design-group10/blob/master/demos/tutorials/yw_save_documentation.md

Reply:
or maybe get rid of the other version that i wrote up all together, not sure which one would be better